README.md

```diff
@@ -87,18 +87,16 @@ $ solana delegate-stake ~/validator-stake-keypair.json ~/validator-vote-keypair.
 More information about staking can be found at https://solana-labs.github.io/book-edge/validator-stake.html
 
 ## Publishing Information About Your Validator
-See https://solana-labs.github.io/book-edge/validator-info.html for background,
-to operate `solana-validator-info` on the TdS cluster you need to include the
-`-u http://tds.solana.com:8899` argument:
+See https://solana-labs.github.io/book-edge/validator-info.html for background:
 
 Example publish command:
 ```bash
-$ solana-validator-info publish -u http://tds.solana.com:8899 ~/validator-keypair.json ...
+$ solana validator-info publish ~/validator-keypair.json mvines -n mvines
```

Comment:
Why u pick on me? :)

How about “elvis” instead of “mvines”

Reply:
I like it! Will change.
I only picked you because you already picked yourself for the `get` example ;)